tests/src/com/unciv/testing/BasicTests.kt

```diff
@@ -134,8 +134,8 @@ class BasicTests {
                 println("${uniqueType.name}'s deprecation text does not match any existing type!'")
                 allOK = false
             }
-            if (replacementTextUnique.getDeprecationAnnotation() != null){
-                println("${uniqueType.name}'s deprecation text references another deprecated unique!'")
```

Comment:
One thing to note is that it could then be the case (if I read the replacing code correctly) that a deprecated unique is upgraded to another deprecated unique, requiring the auto unique updater to be run again, which may not be intuitive for the user

Reply:
I agree but the alternative is that each Unique update requires updating the deprecation text of all the tail, which is considerable more work than pushing a button several times

Reply:
It is indeed possible and that's a good point :)

Reply:
I hadn't approved this yet waiting for this to be added, but I suppose that can be a different PR